server/api/cart.js

```diff
@@ -1,40 +1,56 @@
-const router = require('express').Router();
+const router = require("express").Router();
 const User = require("../db/models/User");
-const Order = require("../db/models/Order");
-// const Category = require("../db/models/Category");
-// const LineItem = require("../db/models/Lineitem");
-// const Product = require("../db/models/Product");
 
 // GET /api/cart
 router.get("/", async (req, res, next) => {
   //find user by token and get their cart
   try {
     const user = await User.findByToken(req.headers.authorization);
+    // const cart = await user.getCart();
+    // console.log("CART WORKS", cart);
     res.send(await user.getCart());
   } catch (error) {
     next(error);
   }
 });
 
-// POST /api/addToCart
-router.post("/addToCart", async (req, res, next) => {
+// POST (Add To Cart) /api/cart
+router.post("/", async (req, res, next) => {
   try {
     const user = await User.findByToken(req.headers.authorization);
     res.send(await user.addToCart(req.body)); //req.body = product
+    // console.log(`req.body`, req.body);
   } catch (error) {
     next(error);
   }
 });
 
-// POST /api/removeFromCart
-router.post("/removeFromCart", async (req, res, next) => {
+// DELETE (Remove From Cart) /api/cart
+router.post("/remove", async (req, res, next) => {
```

Comment:
This should probably be a router.delete to remain RESTful (DELETE request because you're deleting an entry, vs. POST which is supposed to be used for creating entries)